server/applicationPattern/applicationPattern/onfModel/services/models/logicalTerminationPoint/ConfigurationInputV2.js

```diff
@@ -0,0 +1,50 @@
+/**
+ * @deprecated please use ConfigurationInputV2.js
+ * This class provides a stub for the logical-termination-point configuration input.
```

Comment:
@deprecated please use ConfigurationInputV2.js - is present under ConfigurationInputV2 itself. I think it shall be present inside ConfigurationInput.js. Kindly please check

Reply:
Thanks for mentioning. Yes, this deprecated statement should present in the configurationInput.js.